code/controller/SubsiteXHRController.php

```diff
@@ -1,20 +1,36 @@
 <?php
 
+namespace SilverStripe\Subsites\Controller;
+
+
+use SilverStripe\Admin\LeftAndMain;
+use SilverStripe\Security\Permission;
+use SilverStripe\Subsites\Model\Subsite;
+
+
 /**
  * Section-agnostic PJAX controller.
  */
 class SubsiteXHRController extends LeftAndMain
 {
     /**
+     * @todo Temporary addition due to new requirements for LeftAndMain
+     *       descendants in SS4. Consider alternate implementation.
+     */
+    private static $url_segment = 'subsite_xhr';
```

Comment:
See note above on `admin/subsite_xhr`. I suggest making this url_segment the canonical URL rather than the temp one.